testapp/src/main/res/values/strings.xml

```diff
@@ -25,6 +25,9 @@
     <string name="hint_messaging_unique_id_error">Messaging Unique ID Error Message</string>
     <string name="hint_mauid_error">MAUID Error Message</string>
 
+    <string name="hint_set_max_storage_limit">Upgrade Max Storage Limit from 5MB</string>
```

Comment:
```suggestion
    <string name="hint_set_max_storage_limit">Max Storage Limit upto 5MB</string>
```

Also, maybe we can show the already cached value in this field as a hint.

Reply:
Addressed.